demo/admin/src/products/ProductsGrid.tsx

```diff
@@ -48,7 +50,10 @@ import {
 } from "./ProductsGrid.generated";
 import { ProductsGridPreviewAction } from "./ProductsGridPreviewAction";
 
-function ProductsGridToolbar({ exportApi }: { exportApi: ExportApi }) {
+function ProductsGridToolbar({ exportApi, selectionModel }: { exportApi: ExportApi; selectionModel: GridSelectionModel }) {
+    const client = useApolloClient();
+    const theme = useTheme();
```

Comment:
@nsams @Ben-Ho this will be a problem in the Admin Generator: how would I get this code into the toolbar? How could we solve this differently? IMO adding a custom component just for these hooks is an overkill.

Reply:
for now we will have to create a custom component when using admin generator